js/core/player.js

```diff
@@ -86,28 +86,28 @@ define(["dao", "globals", "core/finances", "data/injuries", "data/names", "lib/b
         };
 
         // These use the same formulas as the composite rating definitions in core.game!
-        if (hasSkill(ratings, ['hgt', 'tp'], [0.2, 1])) {
```

Comment:
This could probably be condensed if you make an array of objects containing the skill code and the name of the compositeWeights entry. You don't have to do that though, since your pull request is already better than the status quo.